src/components/global/FilterByChannels.tsx

```diff
@@ -32,7 +30,7 @@ const FilterByChannels = ({
   const [guild, setGuild] = useState<IGuild>();
   const [channels, setChannels] = useState<Array<IGuildChannels>>([]);
   const [selectedChannels, setSelectedChannels] = useState<
-    { channelId: string; channelName: string }[]
+    Array<IChannelWithoutId>
```

Comment:
Nice! 👍🏽 

Does this ever include an `IChannel`?

Reply:
Yes.